tests/components/zwave_js/test_light.py

```diff
@@ -139,6 +172,42 @@ async def test_light(hass, client, bulb_6_multi_color, integration):
 
     client.async_send_command.reset_mock()
 
+    # Test turning on with brightness and transition
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {
+            "entity_id": BULB_6_MULTI_COLOR_LIGHT_ENTITY,
+            ATTR_BRIGHTNESS: 129,
+            ATTR_TRANSITION: 20,
+        },
+        blocking=True,
+    )
+
+    assert len(client.async_send_command.call_args_list) == 1
+    args = client.async_send_command.call_args[0][0]
+    assert args["command"] == "node.set_value"
+    assert args["nodeId"] == 39
+    assert args["valueId"] == {
+        "commandClassName": "Multilevel Switch",
+        "commandClass": 38,
+        "endpoint": 0,
+        "property": "targetValue",
+        "propertyName": "targetValue",
+        "metadata": {
+            "label": "Target value",
+            "max": 99,
+            "min": 0,
+            "type": "number",
+            "readable": True,
+            "writeable": True,
+            "label": "Target value",
+        },
+    }
+    assert args["value"] == 50
```

Comment:
We need to assert the transition update somehow.

Reply:
I didn't see a way to assert it.

Reply:
Shouldn't there be a `args["options"]` with the duration?

Reply:
Good point